content/400-reference/300-database-reference/02-connection-urls.mdx

```diff
@@ -70,9 +70,23 @@ datasource db {
 }
 ```
 
-You can then either set the environment variable in your terminal or by providing a [dotenv](https://github.com/motdotla/dotenv) file called `.env`. This will automatically be picked up by the Prisma CLI.
+### MongoDB (Preview)
```

Comment:
I think the section before this is not actually SQLite specific, but just was missing a proper headline previously which is why now "MongoDB" snuck in between.